recipes/fairlearn/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "0.4.0" %}
```

Comment:
Does that mean we have to keep updating this with every new version?

Reply:
a bot does that automatically whenever you publish a new release on github, for example: https://github.com/conda-forge/scikit-learn-feedstock/pull/110

Or at least, I know there is a way for it to do that :D